src/components/atoms/InputField/InputField.scss

```diff
@@ -0,0 +1,51 @@
+@import "scss/constants";
+
+$icon-width: 2.5rem;
+
+.input-field {
+  position: relative;
+
+  &--icon-start {
+    input {
+      padding-left: $icon-width;
+    }
```

Comment:
can you please assign styles to classes, not tags? It makes the behavior of the component more predictable.

Reply:
I can. :ok_hand: 
I did that because I wanted the selector to be less specified so that it could be easier to overwrite if needed. 
`className + tagName` has lower specificity than `className + className`.

But since `className + tagName` can't be overwritten by a single `className`, and we'll need a `className` or a `tagName` in addition anyway, it seems it doesn't make much sense to think about it here.

Reply:
fixed: https://github.com/sparkletown/sparkle/pull/1306/commits/70022e4294d880248ca128366bff475348bf4ab8